hdr_schemata/utils/create_markdown.py

```diff
@@ -210,14 +214,13 @@ def remove_types(data):
     form["schema_fields"] = []
     form_structure(structure, form)
     with open(f"{path}/{name}.form.json", "w") as f:
-        json.dump(form, f, indent=6)
+        json.dump(clean_markdown_from_json(form), f, indent=6)
```

Comment:
We might not want to clean the markdown for the form structure.  The `3.0.0.form.json`, which is generated in this line, is the file that gets used by the front end for metadata onboarding form hydration. I assume we want the markdown formatting on that form.  
```suggestion
        json.dump(form, f, indent=6)
```

Reply:
Good spot, I thought this was used on the page, I will remove this

Reply:
This should be resolved in the latest commit @BSnelling, will keep an eye out for future